.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
   - repo: https://github.com/psf/black
```

Comment:
https://black.readthedocs.io/en/latest/change_log.html#integrations

Worth switching to the mirror for a 2x speed up?

`https://github.com/psf/black-pre-commit-mirror`

Reply:
Good catch :+1: Changed.